docs/concepts/models/overview.md

```diff
@@ -192,7 +192,7 @@ Name is ***required*** and must be ***unique***.
 - Storage format is a property for engines such as Spark or Hive that support storage formats such as  `parquet` and `orc`.
 
 ### partitioned_by
-- Partitioned by is an optional property for engines such as Spark or BigQuery that support partitioning. Use this to specify a multi-column partition key or to modify a date column for partitioning. For example, in BigQuery you could partition by day by extracting the day component of a timestamp column `event_ts` with `partitioned_by TIMESTAMP_TRUNC(event_ts, DAY)`.
+- Partitioned by plays two roles. For the [`INCREMENTAL_BY_PARTITION` model kind](./model_kinds.md#incremental_by_partition), it defines the partition key used to incrementally load data. For all other model kinds, it is an optional property for engines that support table partitioning such as Spark or BigQuery. Use it to specify a multi-column partition key or to modify a date column for partitioning. For example, in BigQuery you could partition by day by extracting the day component of a timestamp column `event_ts` with `partitioned_by TIMESTAMP_TRUNC(event_ts, DAY)`.
```

Comment:
[Nit] I'd mention `INCREMENTAL_BY_PARTITION` case as a second role because it's less common compared to the other one.